src/upload/catalog/controller/extension/payment/libwebpay_rest/HealthCheck.php

```diff
@@ -130,81 +103,74 @@ private function getPluginLastVersion($ecommerce, $currentversion){
-        return $this->resExtensions;
+        return $extensions;
     }
 
     // crea resumen de informacion del servidor. NO incluye a PHP info
-    private function getServerResume() {
-        // arma array de despliegue
-        $this->resume = array(
+    private function getServerResume(): array {
+        return [
             'php_version' => $this->getValidatephp(),
-            'server_version' => array('server_software' => $_SERVER['SERVER_SOFTWARE']),
+            'server_version' => ['server_software' => $_SERVER['SERVER_SOFTWARE']],
             'plugin_info' => $this->getPluginInfo($this->ecommerce)
-        );
-        return $this->resume;
+        ];
     }
 
     // crea array con la informacion de comercio para posteriormente exportarla via json
-    private function getCommerceInfo() {
-        $result = array(
-            'environment' => $this->environment,
-            'commerce_code' => $this->commerceCode,
-            'api_key' => $this->apiKey,
-        );
-        return array('data' => $result);
+    private function getCommerceInfo(): array {
+        return [
+            'data' => [
+                'environment' => $this->environment,
+                'commerce_code' => $this->commerceCode,
+                'api_key' => $this->apiKey,
+            ]
+        ];
     }
 
     // guarda en array informacion de funcion phpinfo
-    private function getPhpInfo(){
+    private function getPhpInfo(): array {
         ob_start();
         phpinfo();
-        $info = ob_get_contents();
-        ob_end_clean();
+        $info = ob_get_clean();
         $newinfo = strstr($info, '<table>');
-        $newinfo = strstr($newinfo, '<h1>PHP Credits</h1>',true);
-        $return = array('string' => array('content' => str_replace('</div></body></html>','', $newinfo)));
-        return $return;
+        $newinfo = strstr($newinfo, '<h1>PHP Credits</h1>', true);
+        return [
+            'string' => ['content' => str_replace('</div></body></html>', '', $newinfo)]
+        ];
     }
 
-    public function setInitTransaction(){
+    public function setInitTransaction(): array {
         $transbankSdkWebpay = new TransbankSdkWebpay($this->config, new LogHandler());
         $amount = 990;
         $buyOrder = "_Healthcheck_";
         $sessionId = uniqid();
         $returnUrl = "https://webpay3gint.transbank.cl/filtroUnificado/initTransaction";
         $result = $transbankSdkWebpay->initTransaction($amount, $sessionId, $buyOrder, $returnUrl);
-        if ($result) {
-            if (!empty($result["error"]) && isset($result["error"])) {
-                $status = 'Error';
-            } else {
-                $status = 'OK';
-            }
-        } else {
-            if (array_key_exists('error', $result)) {
-                $status =  "Error";
-            }
+
+        $status = (!empty($result["error"]) && isset($result["error"])) ? 'Error' : 'OK';
+        if (!$result && array_key_exists('error', $result)) {
```

Comment:
this is redundant. If `$result` is `false` then can't be an array.
Anyway, are both assessments necessary for `$status`?